src/frontend/frontend.ts

```diff
@@ -1,31 +1,27 @@
-import {Component, Inject, bind, NgZone} from '@angular/core';
-
+import {Component, NgZone} from '@angular/core';
 import {bootstrap} from '@angular/platform-browser-dynamic';
-
 import {Dispatcher} from './dispatcher/dispatcher';
-
 import {BackendActions} from './actions/backend-actions/backend-actions';
 import {UserActions} from './actions/user-actions/user-actions';
-
-import {UserActionType}
-  from './actions/action-constants';
-
-import {ComponentDataStore}
-  from './stores/component-data/component-data-store';
-
+import {UserActionType} from './actions/action-constants';
+import {ComponentDataStore} from './stores/component-data/component-data-store';
 import {BackendMessagingService} from './channel/backend-messaging-service';
-
 import {TreeView} from './components/tree-view/tree-view';
 import {InfoPanel} from './components/info-panel/info-panel';
 import AppTrees from './components/app-trees/app-trees';
 import {Header} from './components/header/header';
-
 import * as Rx from 'rxjs';
-
 import {ParseUtils} from './utils/parse-utils';
 
 const BASE_STYLES = require('!style!css!postcss!../styles/app.css');
 
+// (ericjim) tweak these values based off research on the stress tester.
+const NODE_BREAKPOINT = 1000; // limit at which maximum depth takes effect.
```

Comment:
We guess that if there are 1000 nodes in the angular 2 app, then it must be a pretty complex app. At which point we limit how many depths Augury will open initially. In this case it will stop at a depth of 3. The user can continue revealing nodes at their discretion.